build/mtaircompressor/Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG cycle
+ARG hub
+
+FROM ${hub}/deploy-env:${cycle}
+
+LABEL maintainer Petr Kubánek <pkubanek@lsst.org>
+
+ARG ts_mtaircompressor
+
+RUN source /home/saluser/.setup_sal_env.sh && \
+    conda install -c lsstts ts-mtaircompressor=${ts_adamSensors}
```

Comment:
`ts_adamSensors` -> `ts_mtaircompressor`

Reply:
Fixed, thanks for pointing that out.